docs/manuals/source/TasksAndConcepts/AutochangerSupport.rst

```diff
@@ -505,6 +505,12 @@ Tapespeed and blocksizes
 :index:`\ <single: Tape; speed>`
 :index:`\ <single: Blocksize; optimize>`
 
+.. note::
+  As of Bareos 23, the default block size has been increased to 1 MiB (1.048.576 bytes).
+  This provides optimal tape performance in the default configuration, so there is no need to tune the block sizes anymore.
```

Comment:
```suggestion
  This provides close to optimal tape performance in the default configuration, so there is probably no need to tune the block sizes anymore.
```

Reply:
You're probably right, but that might sounds a bit too vague.
What about "This should provide optimal tape performance in the default configuration and remedy the need to tune block sizes."?

Reply:
Agreed!